File: src/codes/Users_Manual/link/visualizer_matplotlib.cpp
//~#include<autd3.hpp>
#include "autd3/link/visualizer.hpp"

//~int main() {
//~auto autd = autd3::ControllerBuilder()
//~                .add_device(autd3::AUTD3(autd3::Vector3::Zero()))
//~                .open(
autd3::link::Visualizer::builder().with_backend<autd3::link::PythonBackend>()
    //~                    );
    //~return 0; }

File: src/codes/Users_Manual/link/visualizer_matplotlib.rs
# use autd3::prelude::*;
# use autd3_link_visualizer::{Visualizer, PlotConfig, PythonBackend};
# #[allow(unused_variables)]
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::builder([AUTD3::new(Vector3::zeros())])
#     .open(
Visualizer::builder().with_backend::<PythonBackend>()
# ).await?;
# autd.close().await?;
# Ok(())
# }